.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +0,0 @@
----
```

Comment:
Any reason to remove all of the templates?

Reply:
Cause they are templates and should not be included in the tracked git